jiant/tasks/lm_parsing.py

```diff
@@ -32,16 +32,16 @@ def count_examples(self):
             example_counts[split] = int(math.ceil(allf / self.max_seq_len))
         self.example_counts = example_counts
 
-    def process_split(self, split, indexers, boundary_token_fn) -> Iterable[Type[Instance]]:
+    def process_split(self, split, indexers, task_modulator) -> Iterable[Type[Instance]]:
         """Process a language modeling split by indexing and creating fields.
         Args:
             split: (list) a single list of sentences
             indexers: (Indexer object) indexer to index input words
-            boundary_token_fn: Inserts start and end symbols for classification tasks.
-              Not used here. This may be a problem for GPT-2 or future LMs that use non-standard
-              boundary tokens.
+            task_modulator.boundary_token_fn: Inserts start and end symbols for classification
```

Comment:
Can you explain what the "not used here" means? Not very clear from first glance. Also, for documentation, you should put task_modulator: TaskModulator object, with boundary_token_fn function that inserts.....

Reply:
I don't really know. "Not used here" was here before I edited the task.

Reply:
I don't really know. "Not used here" was here before I edited the task.